examples/40_advanced/example_single_configuration.py

```diff
@@ -71,7 +67,7 @@
                                                                 configuration=configuration,
                                                                 budget_type='epochs',
                                                                 budget=20,
-                                                                run_time_limit_secs=70
+                                                                run_time_limit_secs=100
```

Comment:
Why was this increased?

Reply:
because the configuration did not finish on time. As its a random configuration, its better to take an upper bound